src/main/java/amadeus/brain/Database.java

```
@@ -0,0 +1,140 @@
+                    String from = parts[3]; // Start time
+                    String to = parts[4]; // End time
+                    return new Event(description, isDone, from, to);
+
+                case "T": // ToDo
+                    return new ToDo(description, isDone);
+
+                default: // Invalid Saved Task Type Exception
+                    throw AmadeusException.invalidTaskType(type);
+            }
+        } catch (Exception e) {
+            // Error parsing a Saved Task
+            throw AmadeusException.errorParsingTask(line);
+        }
+    }
+
+
+    /**
+     * Loads tasks from the file and returns them as a list of `Task` objects.
+     * If the file does not exist, an empty list is returned. If the file exists,
+     * each line is parsed into a `Task` object using the `parseSave` method.
+     *
+     * @return A list of tasks loaded from the file.
+     * @throws AmadeusException If an error occurs while loading tasks from the file.
+     */
+    public static ArrayList<Task> load() throws AmadeusException
+    {
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        try {
+            // Open the file for reading
+            File file = new File(FILE_PATH);
+            Scanner scanner = new Scanner(file);
+
+            while (scanner.hasNextLine())
+            {
+                String line = scanner.nextLine(); // Read each line
+                Task task = parseSave(line); // parse into Task object
+                tasks.add(task); // Add to Task List
+            }
+
+            // Close the file
+            scanner.close();
+
+        } catch (FileNotFoundException e) { // If the file is not found, start with an empty task list
+            System.out.println("⚠️ No existing task file found. Starting with an empty task list.");
+
+        } catch (Exception e) {
+            // Error Loading Task List from Memory.txt
+            throw AmadeusException.errorLoadingTask(e.getMessage());
+        }
```

Try not to simply catch all exceptions like this instead use the specific exception that may be thrown